pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -107,9 +107,12 @@ while [ "$#" -gt 0 ]; do
         ;;
       --flake)
         flake="$1"
-        flakeFlags=(--extra-experimental-features 'nix-command flakes')
+        flakeFlags+=(--extra-experimental-features 'nix-command flakes')
         shift 1
         ;;
+      -L)
```

Comment:
this is already matched further up, a `--show-trace`

Reply:
Oh, you're absolutely right, I didn't notice that because of the long line :sweat_smile: